src/Raven.Server/Config/Categories/MonitoringConfiguration.cs

```diff
@@ -30,18 +32,114 @@ public sealed class MonitoringConfiguration : ConfigurationCategory
         public MonitoringConfiguration()
         {
             Snmp = new SnmpConfiguration();
+            OpenTelemetry = new OpenTelemetryConfiguration();
         }
 
         public SnmpConfiguration Snmp { get; }
+        public OpenTelemetryConfiguration OpenTelemetry { get; }
 
         public override void Initialize(IConfigurationRoot settings, HashSet<string> settingsNames, IConfigurationRoot serverWideSettings, HashSet<string> serverWideSettingsNames, ResourceType type, string resourceName)
         {
             base.Initialize(settings, settingsNames, serverWideSettings, serverWideSettingsNames, type, resourceName);
             Snmp.Initialize(settings, settingsNames, serverWideSettings, serverWideSettingsNames, type, resourceName);
-
+            OpenTelemetry.Initialize(settings, settingsNames, serverWideSettings, serverWideSettingsNames, type, resourceName);
             Initialized = true;
         }
 
+        [ConfigurationCategory(ConfigurationCategoryType.Monitoring)]
+        public sealed class OpenTelemetryConfiguration : ConfigurationCategory
+        {
+            [Description("Indicates if OpenTelemetry is enabled or not. Default: false")]
+            [DefaultValue(false)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.Enabled", ConfigurationEntryScope.ServerWideOnly)]
+            public bool Enabled { get; set; }
+            
+            [Description("Indicates if server-wide OpenTelemetry is enabled or not. Default: true")]
+            [DefaultValue(true)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.Meters.Server.Enabled", ConfigurationEntryScope.ServerWideOnly)]
+            public bool ServerWideEnabled { get; set; }
```

Comment:
You changes the config option ServerWide -> Server but the property is still ServerWide

Reply:
Below options have `".Instrumentation."` should we add it here as well for consistency or maybe drop it in below options?

Reply:
Changed